src/test/java/org/jabref/gui/groups/GroupTreeViewModelTest.java

```diff
@@ -40,12 +40,12 @@ void setUp() throws Exception {
     @Test
     void rootGroupIsAllEntriesByDefault() throws Exception {
         AllEntriesGroup allEntriesGroup = new AllEntriesGroup("All entries");
-        assertEquals(new GroupNodeViewModel(databaseContext, stateManager, taskExecutor, allEntriesGroup, new CustomLocalDragboard(), mock(PreferencesService.class)), groupTree.rootGroupProperty().getValue());
+        assertEquals(new GroupNodeViewModel(databaseContext, stateManager, taskExecutor, allEntriesGroup, new CustomLocalDragboard(), mock(PreferencesService.class)), groupTree.rootGroupProperty().getValue(), "New group view model is not set to all entries by default.");
```

Comment:
I'm not sure about our test policy @JabRef/developers here? Is it sufficient to express the meaning of the test in the method name or do we additionally output it in the assert clause? What is the suggested standard behavior of junit?

Reply:
Assert messages should only be used in rare cases and never to explain the purpose of the test (that should be done via the test method name). They should only be used when the junit message is not enough to see where the problem is if the assertion fails. For example, you compare two dates expressed in milliseconds, then junit tells you that 53432423 is not 2323232 - not very helpful. In this case, one can supplement it by saying "date is not first of January 2020".